### PR TITLE
Add IOC management example

### PR DESCRIPTION
**Pull request by kpham-polb**
Add a working example to manage IOCs from the CLI. (retrieving IOCs now supports pagination)

### CODE REVIEW

File: examples/edit_custom_iocs/crowdstrike-ioc.go [hunk 1]
```diff
@@ -19,16 +19,29 @@ import (
...
-	}
 	iocs := []string{}
-	for _, ioc := range res2.GetPayload().Resources {
-		iocs = append(iocs, ioc.Value)
+	var limit, offset, total int64
+	limit = 2000
+	offset = 0
+	total = 0
+
+	for offset >= total {
```
Comment (isimluk) [inline examples/edit_custom_iocs/crowdstrike-ioc.go:28]: There seems to be a problem in the pagination logic.

I have environment with only one IOC, but this loop never ends as the offset is always greater than the total.

Comment (isimluk) [reply examples/edit_custom_iocs/crowdstrike-ioc.go:28]: There is already one example that deals with pagination. (You don't need to copy that solution, just mentioning it for the reference).

https://github.com/CrowdStrike/gofalcon/blob/main/examples/falcon_cleanup_pods/main.go#L136-L161 

In this example there is go channel established and asynchronous go routine fills in the channel with the data from API. That way the caller doesn't need to wait for all the items to be fetched from the api can can process the items as they appear in the channel. That is theoretically the ultimate solution for the pagination, but your approach is fine as well, really mentioning this just for reference.

**Comment by isimluk** [comment]
Thank You @kpham-polb for your contribution. Once the pagination logic is fixed we can merge this to main.

I really like your example and I would like to include it in the gofalcon release from now on. I hope you don't mind, if I rename your example from crowdstrike-ioc to falcon_ioc to follow the current naming convention of the tools that are packaged in the release (https://github.com/CrowdStrike/gofalcon/releases )

File: examples/edit_custom_iocs/crowdstrike-ioc.go (hunk 1 above)
Comment (isimluk) [reply examples/edit_custom_iocs/crowdstrike-ioc.go:28]: The following patch fixes pagination for me.
```
-	for offset >= total {
+	for offset <= total {
```

**Comment by kpham-polb** [comment]
Thank you for double-checking my work. I must have copied over the non-working code from another project and committed it, my bad. I've fixed the logic and renamed according to the convention.

**Comment by isimluk** [comment]
This is amazing. Thank You!

**Comment by shawndwells** [comment]
+1. Noticed this appears to be @kpham-polb 's first PR on their GitHub account. Welcome, and thanks for contributing!

**Pull request merged**